app/domain/env.ts

```diff
@@ -20,10 +20,8 @@ export const PUBLIC_URL = (
   ""
 ).replace(/\/$/, "");
 
-export const SPARQL_ENDPOINT =
-  clientEnv?.SPARQL_ENDPOINT ??
-  process.env.SPARQL_ENDPOINT ??
-  "https://int.lindas.admin.ch/query";
+export const SPARQL_ENDPOINT = "https://int.lindas.admin.ch/query";
+export const SPARQL_GEO_ENDPOINT = "https://ld.geo.admin.ch/query";
```

Comment:
This should properly be read from the clientEnv/process.env like the SPARQL_ENDPOINT above.